endpointmanager/pkg/versionsoperatorparser/versionsresponse.go

```diff
@@ -0,0 +1,53 @@
+package versionsoperatorparser
+
+import (
+	"bytes"
+	"encoding/json"
+)
+
+// VersionsResponse is a wrapper struct for a response to the $versions FHIR operation
+// this implementation assumes that we are requesting and receiving the application/json response
+// future support for the FHIR representation of this will require making the VersionsResponse an
+// interface follo
+type VersionsResponse struct {
+	Response map[string]interface{}
+}
+
+// Equal checks if the conformance/capability statement is equal to the given conformance/capability statement.
```

Comment:
This commont should say it checks if the version response is equal instead of the conformance/capability statement